### ISSUE TITLE
Setting tls_private_key/tls_certificate per node has no effect

### ISSUE THREAD

**Issue by neirbowj** [munin-monitoring/munin#1179]
**Describe the bug**

The master always tries to load its certificate and key from the locations defined in the global section of the configuration, even when overridden on a per node basis.

**To Reproduce**

1. Add a node using paranoid mode
```
# munin.conf
[Group;host.example.com]
    tls paranoid
    tls_certificate /non/default/location/cert.pem
    tls_private_key /non/default/location/key.pem
    address host.example.com
```
2. Create the key and certificate in the specified location
3. Run munin-update

**Expected behavior**
`tls_private_key` and `tls_certificate`in the per-node config takes precedence over the default/global values.

**Screenshots & Logs**
```
2019/04/18 16:00:19 [WARNING] No key file "/etc/munin/munin.pem". Continuing without private key.
2019/04/18 16:00:19 [WARNING] No certificate file "/etc/munin/munin.pem". Continuing without certificate.
````

**Desktop (please complete the following information):**
 - OS+Distribution Version: RHEL 6.10
 - Munin Version: munin-2.0.43-1.el6.noarch

**Comment by sumpfralle** [comment]
@neirbowj: could you please verify, whether #1184 would solve this issue for you?

**Comment by neirbowj** [comment]
@sumpfralle Yes, this would solve it. Thank you very much.

**Comment by sumpfralle** [comment]
Fixed in 574d102b322b23da72e687c7cbe130d28f8fa8e2.
